### PR TITLE
NXP-29784: enable preview

### PR DESCRIPTION
**Pull request by doulba**

### CODE REVIEW

**Comment by RSalem07** [comment]
for the the first commit `NXP-29784: allow relax mode` I think your update is to prepare / align on lts 2021, right?. Because the commit message is not consistent with the modification that you made on the first commit [Here](https://github.com/nuxeo/nuxeo-retention/pull/6/commits/425346bb77b1be474ebbb2fae7942669fd29de48)

File: ci/jenkinsfiles/common-lib.groovy [hunk 1]
```diff
@@ -97,10 +241,22 @@ String mavenArgs() {
...
 boolean needsSaucelabs() {
   return this.isPullRequest() && pullRequest.labels.contains('saucelabs')
 }
 
+void rolloutStatus(String kind, String name, String timeout, String namespace) {
+  sh "kubectl rollout status ${kind} ${name} --timeout=${timeout} --namespace=${namespace}"
+}
+
 void runBackEndUnitTests() {
   return {
```
Comment (RSalem07) [inline ci/jenkinsfiles/common-lib.groovy:261]: The return type is declared to be `void` but you return an object, perhaps you want to use `def` keyword

**Review by RSalem07 — COMMENTED** [review]
LGTM, 👍 left some comments / suggestions

**Comment by doulba** [comment]
> for the the first commit `NXP-29784: allow relax mode` I think your update is to prepare / align on lts 2021, right?. Because the commit message is not consistent with the modification that you made on the first commit [Here](https://github.com/nuxeo/nuxeo-retention/pull/6/commits/425346bb77b1be474ebbb2fae7942669fd29de48)

I don't remember adding it for the LTS Version.
and we are building the package from a snapshot parent.
We are not able to know in advance what will be the platform version, that's why we are allowing relax mode ( from my point of view).

**Review by RSalem07 — APPROVED** [review]
LGTM 👍

**Pull request merged**